css/_uswds-theme-custom-styles.scss

```diff
@@ -112,10 +101,10 @@ abbr[title="required"] {
   $site-banner-icon-color: "blue-40v";
   $site-banner-action-color: "gray-20";
   @include typeset($site-banner-font-family);
+  background-color: color("ink");
```

Comment:
They'll be a number of this type of change throughout, changing the order of rules to avoid splitting across nested rules...